site/contents/write_ups/2021/11-08-21_writeup.md

```diff
@@ -0,0 +1,119 @@
+
+The author recalls her drive through a desert valley in Nevada, observing the mining boon in the area. The area is a hotbed for Lithium mining, a material that is a key in the manufacture of rechargeable batteries. Whilst such batteries were never designed for being used to power machines as large as a car (with Tesla being the world's leading consumer of lithium-ion batteries), the batteries remain the best option. 
+
+Whilst electric cars are cited as an excellent solution to our ever increasing carbon emissions. However, as the article notes: "the mining, smelting, export, assemblage, and transport of the battery supply chain has a significant negative impact on the environment and, in turn, on the communities affected by its degradation."
+
+This aside, most electricity used to charge these lithium car batteries is taken from the national grid. In the US, less than a fifth of this electricity comes from renewable energy sources.
+
+This brings us to a broader issue with new technology, where subtle linguistics makes us believe that developments are far less physical than they sound. 
+
+The idea of storing data in the "cloud" immediately brings images of an almost ethereal, mushy collection of data in the sky. Indeed, we *up*load data to the cloud, implying something about the trajectory of the data in question. This conventionally sidesteps the much less sheep-shaped reality that the cloud is just a collection of servers somewhere. Indeed the internet itself, despite being touted as "wireless" is anything but (see this cool little [diagram](https://www.submarinecablemap.com/) and this breakdown of [how the internet works](https://tutorial.djangogirls.org/en/how_the_internet_works/)). All of these physical systems involving servers, smartphones and laptops have lithium batteries at their core.
+
+For every second of virtual time, we extract from Earth's geological history, building smartphones and other devices that have lifetimes of a few years, often [artificially reduced](https://www.theguardian.com/technology/2017/dec/29/apple-apologises-for-slowing-older-iphones-battery-performance). Is this practice really sustainable?
+
+## What information or knowledge are we missing in terms of environmental impact of technology that might help us to act on it?
+
+Where is your "vitual" machine? It is a computer, that's over there. Look at it. Until you realise the internet is a physical thing with physical components, you don't think about the resources. The cloud is actually a collection of servers and components somewhere.
+
+Can we blame our misunderstanding on the wording? The "cloud", "virtual" machines etc. Or do we just not care because we can't/don't do the digging? The internet is a series of tubes, not a truck you load everything onto.
```

Comment:
I googled this and found he was [widely ridiculed](https://en.wikipedia.org/wiki/Series_of_tubes) for saying the tube thing 🤣 maybe not our best analogy!
```suggestion
Can we blame our misunderstanding on the wording? The "cloud", "virtual" machines etc. Or do we just not care because we can't/don't do the digging?
```